.vscode/settings.json

```diff
@@ -8,5 +8,7 @@
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
     "typescript.tsc.autoDetect": "off",
     "editor.tabSize": 2,
-    "typescript.tsserver.log": "verbose"
+    "typescript.tsserver.log": "verbose",
+    "typescript.tsdk": "node_modules/typescript/lib",
+    "typescript.enablePromptUseWorkspaceTsdk": true
 }
```

Comment:
I had to set this to be able to reliably run the tests. I might be missing something, but yeah 🤷🏼